typescript/demo/demo.ts

```diff
@@ -104,14 +105,10 @@ async function createAIConfig() {
   };
 
   const aiConfig = AIConfigRuntime.create("demo", "this is a demo AIConfig");
-  const result = await aiConfig.serialize(model, data, "demoPrompt");
+  const prompts: Prompt[] = await aiConfig.serialize(model, data, "demoPrompt");
 
-  if (Array.isArray(result)) {
-    for (const prompt of result) {
-      aiConfig.addPrompt(prompt);
-    }
-  } else {
-    aiConfig.addPrompt(result);
+  for (const prompt of prompts) {
```

Comment:
We should add this as `addPrompts` in the config API as well (in its own PR)

Reply:
Yea that's a good idea